### PR TITLE
Update _lora-regions.mdx: add Brazil, Argentina to ANZ

### PR DESCRIPTION
**Pull request by AndreiRegiani**
image

Source: https://www.thethingsnetwork.org/docs/lorawan/frequencies-by-country/

* `AU915-928`: Brazil, Argentina

### CODE REVIEW

**Comment by garthvh** [comment]
Can you get copy of that doc translated with google translate so we can check the regulations?  This is just the docs, so a protobuf and firmware pull need to be created before this can be merged.

**Comment by caveman99** [comment]
AU915 is in the firmware already. Please don't add redundant region settings. This is just bloat. Also citing the LoraWAN Frequency list is not helping here, since Meshtastic is not LoraWan. Chances are Brazil is using the ITU Region 2 915 MHz ISM band pretty much like the US does. 

We need the regu8latory docs for the contries, with TX limits and potential duty cycle limitations to add a proper region - OR point people to the region to use.

**Comment by AndreiRegiani** [comment]
>AU915 is in the firmware already. Please don't add redundant region settings.

I don't propose adding a new region to the firmware, for us now it's enough to simply expand the documentation so new LATAM users can see right away which region to pick, this is the #1 question asked in the Brazil Meshtastic Telegram group, there's an influx of new users recently.

Brazil (+ Argentina, Chile) uses AU915(ANZ) already as shown here: https://www.thethingsnetwork.org/docs/lorawan/frequencies-by-country/, also licensed radio folk in that group confirmed ANZ is the one that should be used in Brazil.

As for Argentina & Chile I added based on the link above, which seems to be the same for Brazil.

**Comment by pdxlocations** [comment]
> also licensed radio folk in that group confirmed ANZ is the one that should be used in Brazil.

Sorry but without a link to official regulatory documentation this information is only hearsay.  The link you provided states "This is in no way an official document"

I think it would be awesome to include this regional information but we just can't get this stuff wrong.  Does that make sense?

**Comment by AndreiRegiani** [comment]
We should try to bring regional specialists to comment on this PR, meanwhile this is the info I could gather:

## Brazil

Regulator: `ANATEL` / anatel.gov.br
https://en.wikipedia.org/wiki/National_Telecommunications_Agency_(Brazil)

* Documentation PDF: https://www.anatel.gov.br/Portal/verificaDocumentos/documento.asp?numeroPublicacao=347196

Page `256` (printed: 255), paragraph: `5.150`

```
The following frequency bands:
902-928 MHz (Region 2, center frequency 915 MHz)

are also designated for industrial, scientific, and medical (ISM) applications.
```

---

## Argentina

Regulator: `ENACOM` / enacom.gob.ar
https://en.wikipedia.org/wiki/Ente_Nacional_de_Comunicaciones

* Documentation PDF: https://www.enacom.gob.ar/multimedia/normativas/2018/res581MM.pdf

Page `6`

image

**Comment by garthvh** [comment]
@andrekir Thoughts?

**Comment by renatogmac** [comment]
These informations, documents, and arguments, presented by the applicant, regarding the frequency 919MHz (including the QRG 919.875 used by ANZ modem preset) ,comply with the prevailing telecommunications regulations in Brazil for the ISM band, including multimedia traffic meant for personal use applications - limited to 30 dbm irradiation, and, accepting interference from external sources (similar to FCC Class 2 for ISM)

QRV,
73 de PY2UTU (Renato, Brazil GG66oj)

**Comment by andrekir** [comment]
we can't keep adding all 195+ countries into the region description. I understand the value of indicating each country's LoRa region, but perhaps a in separate reference with a link to "LoRa Region by Country" page with Countries listed in alphabetical order like https://www.thethingsnetwork.org/docs/lorawan/frequencies-by-country/?

data would be added and maintained by the local community on a per-request basis.

> * Documentation PDF: https://www.anatel.gov.br/Portal/verificaDocumentos/documento.asp?numeroPublicacao=347196

this is outdated and LoRa is unlicensed radio (aka "Equipamentos de Radiocomunicação de Radiação Restrita"), prefer the online references (non-PDF) as they are always up-to-date and show changes and revocations: https://www.gov.br/anatel/pt-br/regulado/radiofrequencia/radiacao-restrita

TL;DR in Brazil use ANZ (915-928MHz)

**Comment by AndreiRegiani** [comment]
```
ANZ | Australia, New Zealand, Brazil, Argentina
```

* Removed `Chile` to simplify this PR review, also I couldn't easily find up-to-date info on its regulations.

* Kept `Argentina` as the regulations look clear and easier to validate, if not let's remove it as well.

Brazil is the priority for this PR, despite being a major country, the official Meshtastic documentation doesn't mention which region to use during setup, and a Google search does not return any answers either. At the moment new users may be choosing the wrong region (they copy from English-speaking tutorials), and reading 300-page regulations doesn't help either.

I understand the current region table from the docs won't scale for every country, maybe it's time to refactor that. Luckily it seems many countries are compatible with one of the existing firmware region settings so it's more of a matter of documentation.

Also, I suggested adding to the app itself a country label to further ease on-boarding: https://github.com/meshtastic/Meshtastic-Android/issues/987

**Comment by pdxlocations** [comment]
> "LoRa Region by Country" page with Countries listed in alphabetical order

I quite like this idea.  It gives the opportunity to add limitless information while keeping the region code list light.

**Comment by andrekir** [comment]
> > "LoRa Region by Country" page with Countries listed in alphabetical order

> I quite like this idea. It gives the opportunity to add limitless information while keeping the region code list light.

I put together a quick draft for review in #1230

**Comment by pdxlocations** [comment]
#1230 looks fantastic.  @AndreiRegiani does this solve your concern?  Preview here:
https://meshtastic-git-region-by-country-meshtastic.vercel.app/docs/configuration/region-by-country/

**Comment by AndreiRegiani** [comment]
> #1230 looks fantastic. @AndreiRegiani does this solve your concern? Preview here: https://meshtastic-git-region-by-country-meshtastic.vercel.app/docs/configuration/region-by-country/

Looks like a good template, but Brazil is not there. What else is needed?

**Comment by pdxlocations** [comment]
> Looks like a good template, but Brazil is not there. What else is needed?

Right, the new page is a framework for future PR's requesting additions such as Brazil.  Once it is merged, the community can start adding to it.

**Comment by andrekir** [comment]
@AndreiRegiani the new "region by country" page ready, kindly ask you to point these changes to it (docs/configuration/region-by-coutry.mdx).

**Comment by AndreiRegiani** [comment]
New PR adding Brazil to `docs/configuration/region-by-coutry.mdx`: https://github.com/meshtastic/meshtastic/pull/1241
